Source/WebCore/layout/formattingContexts/inline/InlineItem.h

```diff
@@ -69,15 +71,18 @@ class InlineItem {
 
 private:
     friend class InlineItemsBuilder;
+    friend class TextAutospaceAdjuster;
 
     void setBidiLevel(UBiDiLevel bidiLevel) { m_bidiLevel = bidiLevel; }
     void setWidth(InlineLayoutUnit);
+    void setOffset(float);
 
     const Box* m_layoutBox { nullptr };
 
 protected:
     InlineLayoutUnit m_width { };
     unsigned m_length { 0 };
+    float m_offset { 0 };
```

Comment:
please find a less generic name. I don't want people misuse this highly specific member.